draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2008,17 +2032,19 @@ uCDN-A                         dCDN-B                         dCDN-C
 	<c></c>
 	<c>progress, but the CI/T Trigger Command has been</c>
 	<c></c>
-	<c>canceled by the uCDN.</c>
+	<c>cancelled by the uCDN.</c>
 	<c>cancelled</c>
-	<c>The CI/T Trigger Command was canceled by the uCDN.</c>
```

Comment:
There seems to be a real conflation of the the American canceled / canceling with the British cancelled / cancelling.  This problem starts in the original RFC8007.  RFC7322 Section 3.1 states:
>    The RFC publication language is English.  Spelling may be either
>    American or British, as long as an individual document is internally
>    consistent.  Where both American and British English spelling are
>    used within a document or cluster of documents, the text will be
>    modified to be consistent with American English spelling.

The document appears to prefer American spellings, but the previous RFC8007 used "cancelled" as an official Trigger Status.  Since this is already codified into the previous RFC, I tried to make this consistent even though this deviates from the style guide recommendations.